Smartling.API/Audit/AuditApiClient.cs

```diff
@@ -23,16 +22,16 @@ public AuditApiClient(IAuthenticationStrategy auth, string projectId)
       this.projectId = projectId;
     }
 
-    public virtual void Create(AuditLog log)
+    public virtual void Create(T log)
```

Comment:
I added the new `bucket_name` into `IAuditLog`. Please add here validation:
* `time` and `bucket_name` are not empty
* `time` has proper format. I wanted to add by myself but it make sense to cover by tests

Reply:
@dimitrystd Fields moved to the base class, added custom date serializer